src/pages/_app.tsx

```diff
@@ -35,7 +36,16 @@ export default function App({ Component, pageProps }: AppProps) {
  *
  * `_app`에서만 사용되는 컴포넌트이기에, 인라인으로 작성되었습니다.
  */
+let vh = 0;
+
 function Layout({ children }: PropsWithChildren<{}>) {
+  const windowSize = useWindowSize();
+
+  useEffect(() => {
+    vh = window.innerHeight * 0.01;
```

Comment:
```suggestion


function Layout({ children }: PropsWithChildren<{}>) {
  const windowSize = useWindowSize();

  useEffect(() => {
    const vh = window.innerHeight * 0.01;
```
이렇게 선언해도 되지 않을까 생각이 듭니다!! 

더군다나 이렇게 계산을 할 수 있겠군요!!! 또 하나 배워갑니다 👍 👍 👍

Reply:
그... 저렇게 해서 다시 마운트 되면서 재선언 되는 걸 막고 전역으로 선언해서 재사용하려는  의도였는데 이렇게 해볼까요? ㅎㅎㅎ

Reply:
그러게요.. 저도 재선언과 재할당 중에 어떤 것이 리소스가 많이 드는지는 정확히 알지 못해서 .....
일단 작성하신대로 진행해도 전혀 무리없고 오히려 좋아요!!!

Reply:
훔 저도 전역 변수 선언에 대해서는 조심스럽긴 한데.. 일단 let을 컴포넌트 밖으로 빼야지 재선언을 막는 거고, useEffect(()=>{},[윈도우 하이트])로 재할당을 어느 정도 막은 셈이니 괜찮지 않나 싶었거든요 ㅎㅎㅎ 저도 이 부분은 또 고민해볼게요! 감사합니다 :)